contacts/contact.go

```diff
@@ -0,0 +1,17 @@
+package contacts
```

Comment:
This file was originally going to have more logic in it. It seems a bit pointless to have its own file. But... ready for extensibility, I guess.